cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,36 @@
 /// <reference types='cypress' />
-
 describe('Book Store app', () => {
-  before(() => {
-    
+  before(() => { });
+
+
+  it('login', () => {
+    cy.login();
+    cy.visit('/profile');
+    cy.get('#userName-value').should('contain', 'dmukhailo123');
+    cy.url().should('include', '/profile');
   });
 
-  it('', () => {
-    
+  it('add book', () => {
+    cy.viewport(750, 600);
+    cy.login();
+    cy.visit('/profile');
+    cy.contains('.btn', 'Book Store').click();
+    cy.get('[placeholder="Type to search"]').click().type('Speaking JavaScript');
+    cy.contains('O\'Reilly Media');
+    cy.contains('.action-buttons', 'Speaking JavaScrip').click();
+    cy.contains('.btn', 'Add To Your Collection').click();
+    cy.on('window:alert', (str) => {
+      expect(str).to.equal(`Book added to your collection.`)
+    })
   });
-});
+
+  
+
```

Comment:
There are multiple empty lines (34, 35), consider removing extra lines to keep the code clean and readable.